src/components/rating/Rating.tsx

```diff
@@ -52,15 +52,13 @@ const Rating = ({ isResponseGenerated, feedbackId }: RatingProps) => {
     if (feedback.rating === Ratings.NEGATIVE && !feedback.answerQuality) {
       return;
     }
-    const { status, error } = await SupaBaseDatabase.getInstance().addFeedback({
-      ...feedback,
-      feedbackId,
-    });
 
-    if (status >= 200 && status < 300 && !error) {
-      setIsFeedbackSubmitted(true);
-      console.log("Feedback sent successfully");
+    if (isSupabaseInitialized) {
+      await addFeedback(feedback, feedbackId);
+    } else {
+      console.error('Cannot submit rating because supabase is not initialized');
     }
+    setIsFeedbackSubmitted(true);
```

Comment:
We don't want to do this if the db isn't initialised

Reply:
it's the only way that the page closes